src/app/components/onboarding/register/register.component.ts

```diff
@@ -70,6 +76,9 @@ export class RegisterComponent implements OnInit {
   }
 
   submit(): void {
+
+    for (const k in this.divisions) { if (this.divisions[k]) {this.registerFormGroup.value.division.push(k); } }
```

Comment:
Hast du bei dir schon die Prettier extension installiert? Die Formatierung sieht seltsam aus

Reply:
Nein. Gibt es neben lint check noch einen common Code Style der zu befolgen ist -> bitte in DEV Meeting ansprechen